src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementsByClassName('logo')[0];
+
+const promise1 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was resolved!');
+
+  newDiv.className = 'message';
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  logo.addEventListener('click', () => {
+    resolve(body.insertAdjacentElement('afterbegin', newDiv));
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was rejected!');
+
+  newDiv.className = 'message error-message';
+
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  setTimeout(() => {
+    logo.addEventListener('click', () => {
+      reject(body.insertAdjacentElement('afterbegin', newDiv));
+    });
+  }, 3000);
+});
+
+promise1.then((resolve) => resolve);
+promise2.then((reject) => reject);
```

Comment:
Similar to the previous comment, the `then` method is not being used correctly. The `reject` function should handle the rejected value, but it is currently just returning the `reject` function itself. Consider using a function to handle the rejected value instead.